src/docfx/build/link/LinkResolver.cs

```diff
@@ -192,7 +196,7 @@ internal class LinkResolver
                         if (file is null)
                         {
                             // try to resolve with .md for landing page
-                            file = TryResolveRelativePath(referencingFile, $"{path}.md", lookupGitCommits);
+                            file = TryResolveRelativePath(referencingFile, $"{path}.md", lookupGitCommits, true);
```

Comment:
nit: pass contentFallback to be consistent.
```suggestion
                            file = TryResolveRelativePath(referencingFile, $"{path}.md", lookupGitCommits, contentFallback);
```